src/sentry/scripts/tsdb/cmsketch.lua

```diff
@@ -51,11 +47,11 @@ being called.)
 To add two items, "foo" with a score of 1, and "bar" with a score of 2 to two
 sketches with depth 5, width 64 and index capacity of 50:
 
-    EVALSHA $SHA 6 1:c 1:i 1:e 2:c 2:i 2:e INCR 5 64 50 1 foo 2 bar
+    EVALSHA $SHA 4 1:i 1:e 2:i 2:e INCR 5 64 50 1 foo 2 bar
```

Comment:
Just remembered I modeled this after `ZINCRBY` which is `score key […]` but was trying to use it in tests as `key score […]` like `HINCRBY` and got real annoyed